tls/s2n_send.c

```diff
@@ -96,7 +96,13 @@ int s2n_flush(struct s2n_connection *conn, s2n_blocked_status *blocked)
     }
     POSIX_GUARD(s2n_stuffer_rewrite(&conn->out));
 
-    /* If there's an alert pending out, send that */
+    /* If there's an alert pending out, send that alone.
+     *= https://tools.ietf.org/rfc/rfc8446#5.1
+     *# Alert messages (Section 6) MUST NOT be fragmented across records, and
+     *# multiple alert messages MUST NOT be coalesced into a single
+     *# TLSPlaintext record. In other words, a record with an Alert type
+     *# MUST contain exactly one message.
+     */
     if (s2n_stuffer_data_available(&conn->reader_alert_out) == 2) {
```

Comment:
Would it maybe make sense to add a check in `s2n_record_writev` for `S2N_TLS13` and `TLS_ALERT`, that makes sure only 2 bytes of data exists? This might be better than adding the comment everywhere an alert is sent, even if it's only two places.